spec/models/address_spec.rb

```diff
@@ -1,5 +1,47 @@
 require 'rails_helper'
 
 RSpec.describe Address, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  create_user
+
+  it "is valid with first_line, city, state, zip and country" do
+    address = Address.new(line_one: '1510 Blake Street', city: 'Denver', state: 'CO', zip: '80202', country: 'USA', user_id: user_one.id)
+
+    expect(address).to be_valid
+  end
+
+  it "is invalid without first_line" do
+    address = Address.new(city: 'Denver', state: 'CO', zip: '80202', country: 'USA', user_id: user_one.id)
+
+    expect(address).to_not be_valid
+  end
+
+  it "is invalid without city" do
+    address = Address.new(line_one: '1510 Blake Street', state: 'CO', zip: '80202', country: 'USA', user_id: user_one.id)
+
+    expect(address).to_not be_valid
+  end
+
+  it "is invalid without state" do
+    address = Address.new(line_one: '1510 Blake Street', city: 'Denver', zip: '80202', country: 'USA', user_id: user_one.id)
+
+    expect(address).to_not be_valid
+  end
+
+  it "is invalid without zip" do
+    address = Address.new(line_one: '1510 Blake Street', city: 'Denver', state: 'CO', country: 'USA', user_id: user_one.id)
+
+    expect(address).to_not be_valid
+  end
+
+  it "is invalid without country" do
+    address = Address.new(line_one: '1510 Blake Street', city: 'Denver', state: 'CO', zip: '80202', user_id: user_one.id)
```

Comment:
Line is too long. [121/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.